apps/e2e/forms-storybook/src/app/radio-button/radio-button.component.html

```diff
@@ -6,16 +6,14 @@
       headingText="Radio button options"
       stacked="true"
     >
-      <ul class="sky-list-unstyled">
-        <li *ngFor="let option of radioButtonOptions">
```

Comment:
For the e2e tests, let's keep the ones using unordered lists as-is--and add new ones that do not--so we can be sure this isn't a visual breaking change.

Reply:
similarly to how in some of our visual tests we changed only some labels to labelText, i changed one of the sets of radio buttons in the screenshot to be the intended way, and duplicated the easy mode one with the second one being the intended way